### DISCUSSION TITLE
Docker ratelimiting hosted Renovate jobs

### DISCUSSION THREAD

**Discussion by rbtcollins** [renovatebot/renovate#21132] [Request Help]
### I'm using the Mend Renovate hosted app on github.com.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe your problem.

In recent jobs I'm seeing errors like:
```
Artifact update problem

Renovate failed to update an artifact related to this branch. You probably do not want to merge this PR as-is.

♻ Renovate will retry this branch, including artifacts, only when one of the following happens:

    any of the package files in this branch needs updating, or
    the branch becomes conflicted, or
    you click the rebase/retry checkbox if found above, or
    you rename this PR's title to start with "rebase!" to trigger it manually

The artifact failure details are included below:
File name: Cargo.lock

Command failed: docker pull [docker.io/containerbase/sidecar](http://docker.io/containerbase/sidecar)
Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
```

See https://github.com/rbtcollins/effective-limits.rs/pull/41#issuecomment-1483156701

### Relevant debug logs.

Sorry, don't have logs for this, it seems to have expired or something.

**Comment by rarkins** [comment]
We will switch to using GHCR instead of Docker Hub to resolve this

**Reply by viceice** [reply]
the new backend should probably also use `ghcr.io/renovatebot/renovate` image 😅
